appcircle_android_test_report/1.0.4/component.yaml

```diff
@@ -0,0 +1,49 @@
+platform: Android
+buildPlatform: JavaKotlin, Flutter
+displayName: Test Reports
+description: Parse test and coverage results.
+docsUrl: https://docs.appcircle.io/continuous-testing/running-android-unit-tests/#generating-test-report
+requiredComponents: "appcircle_android_unit_test"
+precedingComponents:
+followingComponents: "appcircle_export_build_artifacts"
+webUrl: https://github.com/appcircleio/appcircle-test-report-component
+repoUrl: https://github.com/appcircleio/appcircle-test-report-component.git
+commit: 02cbb2a
```

Comment:
If the PR on the component side has been merged, can you please get the latest commit from the repo for the PREP update?